src/main/java/sx/blah/discord/api/internal/DispatchHandler.java

```diff
@@ -730,7 +730,7 @@ private void reactionAdd(ReactionEventResponse event) {
 					message.getReactions().add(reaction);
 				} else {
 					reaction.getCachedUsers().add(user);
-					reaction.setCount(reaction.getCount() + 1);
+					reaction.setCount(reaction.getCount());
```

Comment:
What's the point of calling setCount at all then if you provide the result of the respective getter as new value